import-scripts/delete_slow_to_terminate_pods.sh

```diff
@@ -183,6 +183,7 @@ MY_FLOCK_FILEPATH="/data/portal-cron/cron-lock/delete_slow_to_terminate_pods.loc
         mv $temp_file $POD_TERMINATING_TIMESTAMP_FILE
     }
 
+    /data/portal-cron/scripts/authenticate_service_account.sh 
```

Comment:
I wonder why we are calling in the scripts subdirectory. The script is commited in portal-configuration repo, so why not /data/portal-cron/git-repos/portal-configuration/... ?